synapse/rest/client/relations.py

```diff
@@ -230,12 +230,9 @@ async def on_GET(
         original_event = await self._event_serializer.serialize_event(
             event, now, bundle_aggregations=False
         )
-        # Similarly, we don't allow relations to be applied to relations, so we
-        # return the original relations without any aggregations on top of them
-        # here.
-        serialized_events = await self._event_serializer.serialize_events(
-            events, now, bundle_aggregations=False
-        )
+        # For any relations applying to the original event they need their
+        # aggregations applied to them.
```

Comment:
I'm a bit confused about what this comment is trying to tell me?

Reply:
I've attempted to clarify it!